isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp

```diff
@@ -1051,10 +1130,38 @@ namespace Isis {
         sprintf(buf, " %12d %12d     ", numUsed, numMeasures);
         fpOut << buf;
 
-        sprintf(buf,"%13.4lf %13.4lf %13.4lf \n",
+        sprintf(buf,"%13.4lf %13.4lf %13.4lf",
                 rmsSampleResiduals,rmsLineResiduals,rmsLandSResiduals);
 
         fpOut << buf;
+
+        if (m_statisticsResults->outputLidarData()) {
+          double rmsLidarSampleResiduals = m_statisticsResults->
+                                          rmsLidarImageSampleResiduals()[imageIndex].Rms();
+          double rmsLidarLineResiduals = m_statisticsResults->
+                                        rmsLidarImageLineResiduals()[imageIndex].Rms();
+          double rmsLidarLandSResiduals =  m_statisticsResults->
+                                          rmsLidarImageResiduals()[imageIndex].Rms();
+
+          numLidarMeasures = m_statisticsResults->outputLidarData()->
+                             GetNumberOfValidMeasuresInImage(bundleImage->serialNumber());
+
+          numLidarRejectedMeasures = m_statisticsResults->outputLidarData()->
+                             GetNumberOfJigsawRejectedMeasuresInImage(bundleImage->serialNumber());
```

Comment:
Will these get the number of valid Lidar points specifically? Or just all measures in the Network?

Reply:
This will only be for LiDAR points because we store the LiDAR points separate from the regular points. For regular points we call `m_statisticsResults->outputControlNet()`, for LiDAR points we call `m_statisticsResults->outputLidarData()`